Formula/asciiquarium.rb

```diff
@@ -53,6 +88,8 @@ def install
   end
 
   test do
+    return if ENV["CI"] # CI environment post-10.14 can't use ptys
```

Comment:
This is a strange situation because the test ran and worked when we added the catalina bottle, and all sandboxing should also happen locally. The issues other formulae are having are usually resource based, not sandbox.

Reply:
Yeah I don't fully understand this either... looking at other formula there are `PTY.spawn`-using formula that have `:big_sur` bottles.  Unless those were overridden they must have worked.

Yet the ones I've been trying today have definitely been failing on 10.15 and 11 every time (but run locally on 11 no problem)  Maybe something regressed in the CI infra itself and pty-using tests that used to pass on 10.15/11 are now failing?

Reply:
We definitely don't want to disable tests, or remove them. The CI should be able to perform terminal-based tests, if we have specific failures we should investigate or open a brew issue.